### PR TITLE
Added text color and font options for native radio and checkbox

### PR DESCRIPTION
**Pull request by zdevwu**

### CODE REVIEW

**Comment by 13r0ck** [comment]
I believe these can be set by putting a radio / checkbox in a container.

File: native/src/widget/radio.rs [hunk 1]
```diff
@@ -196,8 +212,8 @@ where
             label_layout.bounds(),
             &self.label,
             self.text_size.unwrap_or(renderer.default_size()),
-            Default::default(),
-            None,
+            self.font,
+            self.text_color.clone(),
```
Comment (hecrj) [inline native/src/widget/radio.rs:216]: Is calling `clone` necessary here? `Color` implements `Copy`.

**Review by hecrj — CHANGES_REQUESTED** [review]
Thank you! This is great 😄

File: native/src/widget/radio.rs (hunk 1 above)
Comment (zdevwu) [reply native/src/widget/radio.rs]: nope, I just typed unconsciously :)

**Review by hecrj — APPROVED** [review]
Looks great! Thank you 🎉

**Pull request merged**